libr/bin/format/elf/elf.c

```diff
@@ -3028,6 +3028,9 @@ static RBinElfSymbol* get_symbols_from_phdr(ELFOBJ *bin, int type) {
 
 	addr_sym_table = Elf_(r_bin_elf_v2p) (bin, bin->dyn_info.dt_symtab);
 	sym_size = bin->dyn_info.dt_syment;
+	if (sym_size < 1) {
```

Comment:
Can simsize be negative? If so fix it, otherwise just check for == 0

Reply:
i'll fix this since is `ut32`